browser/brave_content_browser_client.cc

```diff
@@ -308,12 +308,13 @@ void BraveContentBrowserClient::MaybeHideReferrer(
           GURL(),
           CONTENT_SETTINGS_TYPE_PLUGINS,
           brave_shields::kBraveShields);
+  const GURL new_referrer_url; // navigations get empty referrers
```

Comment:
nit: It's a good pratice to run linter, e.g. here it could tell that there should be two spaces before `//`
Also it is a good style to write a narrative text, i.e. `Navigations get empty referrers.` (https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar)

Reply:
I think the naming is also confusing here, maybe `empty_referrer_url`?